src/bazel-tsconfig-build.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This needs a comment that explains why bazel needs a separate tsconfig from the main build

Reply:
Done.

Reply:
Why can't this use the existing `tsconfig-build.json` in the root of the cdk?

Reply:
I think it can, but this tsconfig includes unnecessary options (e.g. the flat module id). The tsconfig should be as generic as possible. At some point there should be only two tsconfigs for all packages. Based on my findings, this is the same what Angular core does.

Reply:
I'd rather use the existing config if it works for both. That way we know the code we're testing via bazel is the same as the code we're testing/releasing with the gulp setup

Reply:
As discussed on Slack. The build config now extends the bazel build config. Meaning that both builds should equal. Just that the top-level tsconfig is without CDK specific options (e.g. path mappings; module name). Such specific options will be handled by Bazel automatically.